sdc.py

```diff
@@ -49,18 +50,18 @@
 class StreamsetsDataCollector:
 
     def __init__(self, version, namespace, registry, os_major_version):
-        self.os_major_version = os_major_version
-        self._image_name = IMAGE_NAME_TEMPLATE.format(registry, namespace, version, os_major_version)
-        self.image_name_no_registry = IMAGE_NAME_NO_REGISTRY_TEMPLATE.format(namespace, version, os_major_version)
-        self.namespace = namespace
-        self.registry = registry
-        self.version = version
-        self.version_tuple = tuple(int(i) if i.isdigit() else i for i in version.split('.'))
+        self._os_major_version = os_major_version
+        self._full_image_name = FULL_IMAGE_NAME_TEMPLATE.format(registry, namespace, version, os_major_version)
+        self._image_name = IMAGE_NAME_TEMPLATE.format(namespace, version, os_major_version)
+        self._namespace = namespace
+        self._registry = registry
+        self._version = version
+        self._version_tuple = tuple(int(i) if i.isdigit() else i for i in version.split('.'))
         self._user_libs_exist = False
 
     @property
```

Comment:
If all the property does is return the instance attribute, get rid of the property and just have the instance attribute not be "private" (with the leading underscope). `@property` is more for when there's some kind of business logic to do to an attribute beyond just returning it directly.